src/Compiler/Microsoft.AspNetCore.Razor.Language/test/CodeGeneration/DesignTimeNodeWriterTest.cs

```diff
@@ -457,6 +457,52 @@ Render Children
             ignoreLineEndingDifferences: true);
     }
 
+    [OSSkipConditionTheory(new[] { "Linux", "osx" })]
```

Comment:
Perhaps we could create an inverted attribute for this, like `OSConditionTheory(new[] { "Windows" })`.

Reply:
It'd be nice to generally make that attribute more convenient. Use an "OS" enum instead of strings, make the array argument params, etc.

Reply:
We should just copy over the roslyn utils for this.